src/adapter/base.cr

```diff
@@ -32,7 +37,8 @@ abstract class Granite::Adapter::Base
   end
 
   def log(query : String, params = [] of String) : Nil
-    Granite::ORM.settings.logger.info "#{query}: #{params}"
+    Granite::ORM.settings.logger.info query.gsub(SQL_KEYWORDS_REGEX, "\\0".colorize(:white).mode(:bold).to_s )
```

Comment:
This colorization could probably be done once and saved? Or is the compiler just going to optimize that?

Reply:
Not sure, but I agree maybe we can colorize and cache it